config.toml

```
@@ -0,0 +1,19 @@
+username = "my-user"
+repository = "https://github.com/my-org/my-repo"
+
+[[labelRule]]
+  [labelRule.create]
+  name = "label1"
+  description = "a cool label"
+  hexColor = "#000000"
```

I think the name here can be just `color`. The `hex` part is implicity implied by the `#` in the color.
```suggestion
  color = "#000000"
```